Document/0x05h-Testing-Platform-Interaction.md

```diff
@@ -1646,6 +1680,9 @@ Lastly, see if you can play with the version number of a man-in-the-middled app
 - MSTG-PLATFORM-7: "If native methods of the app are exposed to a WebView, verify that the WebView only renders JavaScript contained within the app package."
 - MSTG-PLATFORM-8: "Object serialization, if any, is implemented using safe serialization APIs."
 - MSTG-PLATFORM-9: "The app does not allow any other application to overlay on itself."
+- MSTG-PLATFORM-10: "A WebView's cache, storage, and loaded resources (JavaScript, etc.) should be cleared before the WebView is destroyed."
+- MSTG-PLATFORM-11: "Verify that the app prevents usage of custom third-party keyboards whenever sensitive data is entered."
+- MSTG-PLATFORM-12: "High confidentiality information is not parsed into notifications."
```

Comment:
You can find the requirements we are referencing in the MASVS here: https://github.com/OWASP/owasp-masvs/blob/master/Document/0x11-V6-Interaction_with_the_environment.md

Please remove this requirement. 

For changes to the requirements please raise an issue. For this case the following requirement might make the most sense, but will discuss with @cpholguera. What do you think?

2.7 | MSTG-STORAGE-7 | No sensitive data, such as passwords or pins, is exposed through the user interface.
-- | -- | --

Reply:
Will do; I wasn't sure if this needed to be modified / changed for new sections like this so I thought it'd be best I just do it anyway as worst comes to worst it can just be removed and changed back (which ill do now).